src/test/java/org/pesmypetcare/webservice/thirdpartyservices/adapters/firestore/FirestoreCollectionAdapterTest.java

```diff
@@ -163,17 +190,17 @@ public void getCollectionGroupDocumentsWhereFieldEqualToValue() {
             given(query.whereEqualTo(same(field), same(value))).willReturn(query);
             Query result = collectionAdapter.getCollectionGroupDocumentsWhereEqualTo(collectionGroup, field, value);
             assertEquals(query, result,
-                "Should return all documents from the collections whose id is " + collectionGroup + " where " + field
-                    + " is equals to " + value);
+                         "Should return all documents from the collections whose id is " + collectionGroup + " where "
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "Should return all documents from the collections whose id is " appears 4 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5465948)